pcdsdevices/utils.py

```diff
@@ -69,3 +71,27 @@ def get_input():
         # Restore the terminal to normal input mode
         termios.tcsetattr(sys.stdin, termios.TCSADRAIN, old_settings)
         return inp
+
+
+def convert_unit(value, unit, new_unit):
+    """
+    One-line unit conversion
+
+    Parameters
+    ----------
+    value: ``float``
+        The starting value for the conversion.
+
+    unit: ``str``
+        The starting unit for the conversion.
+
+    new_unit: ``str``
+        The desired unit for the conversion
+
+    Returns
+    -------
+    new_value: ``float``
+        The starting value, but converted to the new unit.
+    """
+    start_unit = Unit(unit)
```

Comment:
What exception gets raised on a bad unit? A sensical one?

Reply:
fyi this is also throwing `/u1/zlentz/conda/envs/dev/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88` on import because `numpy` had a regression in the warning ignores block. I've looked into it and found that it isn't dangerous, but I might go back and suppress the warning explicitly.